.github/ISSUE_TEMPLATE/membership.yaml

```diff
@@ -0,0 +1,75 @@
+# Modifications Copyright SAP SE or an SAP affiliate company and Gardener contributors.
+name: Membership Request
+description: Request a membership status in the project
+labels: ["kind/task"]
+title: "Membership role <role-name> for <your-GH-handle>"
+body:
+- id: motivation
+  type: textarea
+  attributes:
+    label: "Motivation"
+    description: Describe the motivation of this request and your background shortly - we'd like to know you :-)
+- id: github
+  type: input
+  attributes:
+    label: GitHub Username
+    placeholder: e.g. @example_user
+  validations:
+    required: true
+- id: role
+  type: dropdown
+  attributes:
+    label: "Requested Role"
+    description: Choose the community role you want to apply for as described in [community roles](https://gardener.cloud/docs/contribute/code/roles).
```

Comment:
Is `https://gardener.cloud/docs/contribute/code/roles` the place where the role document is going to be found?
I'm just asking the link does not work yet.

Reply:
Exactly, it'll be available once https://github.com/gardener/documentation/pull/715 is merged.